dandi/models.py

```diff
@@ -563,6 +564,14 @@ class CommonModel(DandiBaseModel):
         None, readOnly=True, nskey="prov"
     )
 
+    def asdict(self):
```

Comment:
I only wonder if it would not bring us confusion as there is `.dict` provided by pydantic as well... may be something like `dict_via_json` or `json_dict`. Although uglier, and not following of `as*` convention - it would be closer to pydantic's `dict`, `json`?

Reply:
Of those suggestions, I prefer `json_dict`, though not as much as I like `asdict`.

Reply:
Let's proceed with `json_dict` then.

Reply:
Renamed.